anom/adapters/datastore_adapter.py

```diff
@@ -234,11 +234,38 @@ def _prepare_to_store(self, key, unindexed, data):
         for name, value in data:
             if isinstance(value, Key):
                 value = self._convert_key_to_datastore(value)
-
-            entity[name] = value
+            if isinstance(value, datastore.Entity) and value.get('structured'):
+                value.pop('structured')
+                self._prepare_structured_to_main(entity, name, value)
+            else:
+                entity[name] = value
 
         return entity
 
+    def _prepare_structured_to_main(self, entity, name, value):
```

Comment:
Hey there, just a random observation from a happy stranger that just discovered this project looking for a sane solution for the app engine flexible: you might want to return a new dict in this method to allow better testing and reduce the call logic :-) (oh, and this method could be static)

but enough of that... I should get back to my work  x)